src/main/java/it/gov/pagopa/debtposition/controller/pd/crud/api/v1/DebtPositionController.java

```diff
@@ -179,13 +182,24 @@ public ResponseEntity<PaymentPositionModelBaseResponse> getOrganizationDebtPosit
         segregationCodes != null
             ? new ArrayList<>(Arrays.asList(segregationCodes.split(",")))
             : null;
+
+    PaymentPosition paymentPosition =
+        paymentPositionService.getDebtPositionByIUPD(organizationFiscalCode, iupd, segCodes);
+
     // flip entity to model
-    PaymentPositionModelBaseResponse paymentPositionResponse =
-        ObjectMapperUtils.map(
-            paymentPositionService.getDebtPositionByIUPD(organizationFiscalCode, iupd, segCodes),
-            PaymentPositionModelBaseResponse.class);
+    PaymentPositionModelBaseResponse basePaymentPositionResponse =
+        ObjectMapperUtils.map(paymentPosition, PaymentPositionModelBaseResponse.class);
```

Comment:
Since `PaymentPositionModelEnhancedResponse` extends `PaymentPositionModelBaseResponse` you could use directly the enhanced version here avoiding using reflection below (the copyProperties) to clone objects and then leave the set additional fields part.
What do you think?